lib/std/sums.nim

```diff
@@ -9,7 +9,7 @@
 ## Fast sumation functions.
 
 
-func sumKbn*[T](x: openArray[T]): T =
+func sumKahan*[T](x: openArray[T]): T {.deprecated: [sumKbn: sumKahan].} =
```

Comment:
put `{.deprecated: [sumKbn: sumKahan].}` after the proc definition, eg:
```nim
func sumKahan*() =
  impl
{.deprecated: [sumKbn: sumKahan].}
```
(that's why CI fails)

Reply:
Thanks, it should be fixed now.